GHzDACs/jumpTable.py

```diff
@@ -189,48 +211,81 @@ class JumpTable(object):
     TODO: make self.jumps a property with some checking on type and number
     """
     PACKET_LEN = 528
-    COUNT_MAX = 2**32 - 1 #32 bit register for counters
-    
-    def __init__(self, startAddr=None, jumps=None):
-        self._counters = [0,0,0,0]        
+    COUNT_MAX = 2 ** 32 - 1  # 32 bit register for counters
+
+    def __init__(self, startAddr=None, jumps=None, counters=None):
+        """
+
+        :param startAddr: start address
+        :param jumps:
+        """
+        self._startAddr = 0
+        if counters is None:
+            self._counters = [0, 0, 0, 0]
+        else:
+            self._counters = list(counters)[0:4]
```

Comment:
`4` here should probably be a numeric constant, and technically it is board-dependent. Does that mean we need to have versioned jump-table implementations, like we do dac classes?